solutions/21.js

```diff
@@ -0,0 +1,7 @@
+const solution = (node, nextNode = node.next, max = node.next) => {
```

Comment:
I'm not sure why max is initialized to node.next. I'm just curious?

Reply:
I have no idea. I think I did that for the sake of giving it a value. I changed it to node.value because that's a smarter move. Thanks!